site/docs/get-started/create-service-item.md

```diff
@@ -74,5 +106,14 @@ Once you are logged out, enter ***https://\[your IP ADDRESS\]\]/self\_service***
 interface, that can be used to isolate you from the nuances of the
 underlying infrastructure.
 
+![screenshot0052](/assets/images/docs/screenshot_0052.png){:.img-responsive}
+
+Self service interface allows an easier management of services and a easier design for end users than the operational interface
```

Comment:
"_The_ Self Service interface allows _for_ easier … and _an_ easier design …" 

or "… a more simple design for end users compared to the operational UI." might read a little better.

Also, isn't supposed to be The _Service UI_ now? (no "self")